.gitignore

```diff
@@ -15,7 +15,12 @@ tags
 /src/tests/**/*.sh
 /src/tests/**/*.php
 /src/tests/**/phpt.*
-
+/src/file_to_include1.php
+/src/file_to_include2.php
+/src/tests/disable_function/test.bla
+/src/tests/disable_function/test.meh
+/src/tests/disable_function/test.sim
+/src/tests/eval_blacklist/test.bla
 
```

Comment:
Those files should be deleted by the testsuite, instead of being ignored here.

Reply:
I tried using `--CLEAN--`, but it seems php does not run the `.clean.php` snippet with the supplied configuration from `--INI--` and thus the test borks, since the cleanup output is not empty but contains `No configuration specified via sp.configuration_file`.